### DISCUSSION TITLE
Ability to embed images into default email notifications

### DISCUSSION THREAD

**Discussion by shalak** [netdata/netdata#14395] [Q&A]
I'm looking for a way, to tune the default email notifications, to make them not use remote content (for netdata logo, etc).

From what I found, the [email notifier plugin](https://github.com/netdata/netdata/blob/master/health/notifications/alarm-notify.sh.in) keeps the whole email content hardcoded in `.sh` file. Is there some supported way to use different templates?

**Comment by andrewm4894** [comment]
hmm - not sure - @MrZammler any idea on this one?
